src/libtomahawk/resolvers/ScriptJob.h

```diff
@@ -64,6 +66,7 @@ public slots:
     QString m_id;
     scriptobject_ptr m_scriptObject;
     QVariantMap m_data;
+    QVariant m_data_primitive;
```

Comment:
So far likely. The idea is that you can still access the data even if you just passed on the ScriptJob to some helper method and don't need to pass the data along simultaneously (because you could only access it in the slot). 

But yeah, it would be good if you could extend the SyncScriptJob in a way that allows to initialize it with a pure QVariant (not Map)